README.md

```diff
@@ -3,37 +3,44 @@ Fill in the text below with your own answers to each question. Add your answers
 
 ## About You
 What are your names, as they appear on the class roster?
-> ...
+> Vasilisa Bashlovkina
+  Jeung Rac Lee
 
 What do you prefer to be called?
-> ...
-
+> Vasilisa or V
+  JR  
+  
 Why are you taking this course?
-> ...
-
+> I like parallel things and graduating on time. - V  
+  It was the only computer science class that fits my schedule. - JR  
+  
 What are your concerns about this course?
-> ...
+> I don't really know anything about operating systems! - V  
+  Most of other classmates seems to be upper class or seniors - JR  
 
 What topic in this course are you most excited to cover?
-> ...
+> I'm excited to learn about parallel algorithms. - V  
+  I'm looking forward to learn about operating system - JR  
 
 Do you have any questions for me?
-> ...
+> When did you start learning about this topic? - V  
```

Comment:
My first OS course was in my third year of college, but I had been playing with some OS-like topics for about a year before that.